packages/protocol/contracts/L1/ProverPool.sol

```diff
@@ -431,7 +434,10 @@ contract ProverPool is EssentialContract, IProverPool {
         return true;
     }
 
-    // Calculates the user weight's when it stakes/unstakes/slashed
+    /// @dev See the documentation for the callers of this internal function.
```

Comment:
Here in this doc, we used /** **/ and ///, shall we stick to one format across all files?